x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/index.tsx

```diff
@@ -647,9 +647,9 @@ export const AgentListPage: React.FunctionComponent<{}> = () => {
         selectionMode={selectionMode}
         currentQuery={kuery}
         selectedAgents={selectedAgents}
-        refreshAgents={({ refreshTags = false }: { refreshTags?: boolean } = {}) =>
-          Promise.all([fetchData({ refreshTags }), refreshUpgrades()])
-        }
+        refreshAgents={({ refreshTags = false }: { refreshTags?: boolean } = {}) => {
+          Promise.all([fetchData({ refreshTags }), refreshUpgrades()]);
```

Comment:
Does it matter that we've changed the return value of this callback? It used to return the Promise containing the results of `fetchData` and `refreshUpgrades` and now it returns a `Promise<void>`. If we don't use the return value from `refreshAgents` I think it's probably fine - just want to clarify.

Reply:
Good catch, the return value is void of this callback, so the value is not used. Though this change is not necessary, I can revert it.
```
refreshAgents: (args?: { refreshTags?: boolean }) => void;
```